examples/conversion_huggingface_models.py

```diff
@@ -0,0 +1,58 @@
+from farm.modeling.adaptive_model import AdaptiveModel
```

Comment:
To make it Windows compatible, we could wrap the example in a method that gets called under `if __name__ == "__main__"` like other examples.